journey-map/public/style.css

```diff
@@ -189,3 +189,11 @@ form fieldset p {
     stroke-width: 3px;
     filter: drop-shadow(1px 0 0 #ffdd00) drop-shadow(-1px 0 0 #ffdd00) drop-shadow(0 1px 0 #ffdd00) drop-shadow(0 -1px 0 #ffdd00);
 }
+#header-actions {
+    position: fixed;
+    right: 30px;
```

Comment:
You can do this more naturally with `margin-left: auto` - that way it'll flow correctly if we start putting other items in the header.